src/styles/main.css

```diff
@@ -0,0 +1,303 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: local("Avenir Book"), local("Avenir-Book"), url("../fonts/book/Avenir-Book.woff2") format("woff2"), url("../fonts/book/Avenir-Book.woff") format("woff"), url("../fonts/book/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: local("Avenir Heavy"), local("Avenir-Heavy"), url("../fonts/book/Avenir-Heavy.woff2") format("woff2"), url("../fonts/book/Avenir-Heavy.woff") format("woff"), url("../fonts/book/Avenir-Heavy.ttf") format("truetype");
+}
+
+body {
+  margin: 0;
+  font-family: "Avenir-Heavy", sans-serif;
+}
+
+.header .logo-bar {
+  padding: 26px 15px 19px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+}
+
+.header .logo-bar .logo {
+  height: 30px;
+  width: 30px;
+}
+
+.header .logo-bar .logo-bar-nav {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+}
+
+.header .logo-bar .logo-bar-nav .logo-bar-image {
+  -o-object-fit: cover;
+     object-fit: cover;
+  height: 30px;
+  width: 30px;
+  border-radius: 50%;
+}
+
+.header .logo-bar .logo-bar-nav a {
+  text-decoration: none;
+  color: #3d4e61;
+  font-style: "Heavy";
+  font-size: 16px;
+  padding-right: 40px;
+}
+
+.header .banner {
+  background-image: url(../images/main-photo.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 464px;
+}
+
+.main {
+  padding: 40px calc(50% - 451px) 25px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+}
+
+.main .apartment {
+  color: #3d4e61;
+  max-width: 492px;
+}
+
+.main .apartment .sec-head {
+  font-size: 36px;
+  line-height: 42px;
+}
+
+.main .apartment .sub-head {
+  font-size: 16px;
+  line-height: 19px;
+  color: #687c94;
+}
+
+.main .apartment .host-button {
+  width: 150px;
+  height: 50px;
+  outline: none;
+  font-size: 16px;
+  color: #118488;
+  background-color: white;
+  border: 1px solid #118488;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  border-radius: 4px;
+}
+
+.main .apartment .apart-details {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.main .apartment .apart-details p {
+  padding-right: 82px;
+  font-size: 16px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.main .apartment .apart-info {
+  font-size: 16px;
+  line-height: 24px;
+  color: #3d4e61;
+}
+
+.main .apartment .contact-host {
+  font-weight: bolder;
+}
+
+.main .apartment .amenity-head {
+  font-weight: bolder;
+}
+
+.main .apartment .facilities span {
+  margin-left: 120px;
+}
+
+.main .apartment .show-all {
+  font-weight: bolder;
+}
+
+.main form {
```

Comment:
will be cool, if you add position sticky to form

read about how it works and how to use it